### ISSUE TITLE
Add git tags visually in the Files and versions tab

### ISSUE THREAD

**Issue by BramVanroy** [huggingface/huggingface_hub#960]
Currently, it is nice to see the commit history of models and datasets. For datasets, it is possible to [load revisions by git tag](https://huggingface.co/docs/datasets/v2.4.0/en/package_reference/loading_methods#datasets.load_dataset.revision). However, in the Commit History overview, these tags are not visible so as a user you are not sure which tag to use (although you can fallback to commits of course).

So in short: it would be nice to see the git tags (if any) next to the commit hash in the Commit History overview.

**Comment by osanseviero** [comment]
Thanks for the idea! I'm linking an internal issue we have for this as well :) https://github.com/huggingface/moon-landing/issues/1387

cc @gary149 @julien-c

**Comment by BramVanroy** [comment]
@osanseviero Oh that is awesome! It seems that none of my ideas are original and are already tracked internally, haha!

**Comment by osanseviero** [comment]
Haha not at all, your ideas are great and aligned with what we want to achieve, which is amazing! Please keep bringing your ideas! 🤗

**Comment by juhoinkinen** [comment]
Any updates on this? I too would like to see the tags along commits. 

Moreover, also _creating_ tags via the website UI or with [huggingface-cli](https://huggingface.co/docs/huggingface_hub/v0.21.4/en/guides/cli) would be nice (just [asked about this](https://discuss.huggingface.co/t/tagging-with-git-tags-models-for-versioning/78489) on the forum).

**Comment by Wauplin** [comment]
Hi @juhoinkinen, no update on this AFAIK. I pinged the internal issue with your message. About creating tags from the CLI I just created https://github.com/huggingface/huggingface_hub/issues/2141 to track it.
